packages/client/src/client.js

```diff
@@ -342,11 +342,8 @@ export class PercyClient {
     validateId('build', buildId);
     if (filepath) {
       content = await fs.promises.readFile(filepath);
-      if (process.env.PERCY_GZIP) {
-        content = Pako.gzip(content);
-      }
     }
-    let encodedContent = base64encode(content);
+    let encodedContent = getEncodedContent(content);
```

Comment:
There is already percy gzip check inside discovery. So this will cause double gzip

Reply:
In network file we are not setting it to encoded content we are just using it for size calculation

Reply:
client code is existing one just moved it to method

Reply:
Should I remove then?

Reply:
No its not :P you missed a condition